lib/dotcom_web/components/system_status/status_row_heading.ex

```diff
@@ -127,17 +128,17 @@ defmodule DotcomWeb.Components.SystemStatus.StatusRowHeading do
   defp description(status, prefix, true), do: description(status, prefix) |> Inflex.pluralize()
   defp description(status, prefix, false), do: description(status, prefix)
 
-  defp description(:normal, _), do: "Normal Service"
-  defp description(:see_alerts, _), do: "See Alerts"
+  defp description(:normal, _), do: ~t"Normal Service"
+  defp description(:see_alerts, _), do: ~t"See Alerts"
 
-  # Special case for delays - when displayed with a future date, say 
+  # Special case for delays - when displayed with a future date, say
   # "Expect Delay" (or Expect Delays) rather than simply "Delay". A
   # prefix of "Now" should still display as "Delay", rather than
   # "Expect Delay".
-  defp description(:delay, "Now"), do: "Delays"
-  defp description(:delay, prefix) when is_binary(prefix), do: "Expect Delays"
-  defp description(:delay, _), do: "Delays"
-  defp description(:shuttle, _), do: "Shuttles"
+  defp description(:delay, "Now"), do: ~t"Delays"
```

Comment:
Once we translate [the string "Now"](https://github.com/mbta/dotcom/blob/9c911bf366b98cda16693ac2eadd3feed1c839ff/lib/dotcom_web/components/system_status/subway_status.ex#L312), this match won't work anymore outside of English, which means that translations will look like `Now: Expect Delays` (e.g. `Ahora: Demoras esperadas up to 10 min`).

I don't think that needs to block this PR, but I did want to flag that.

Reply:
ehhh I think this should be blocking